### ISSUE TITLE
Change name for software from aeon to aeonix or other

### ISSUE THREAD

**Issue by ghost** [aeonix/aeon#210]
It is important to distinguish that this is not AEON (the decentralized digital currency network), this repository is only creating a "browser" for us to connect and send transactions. This is like having a web browser named "Internet".  AEON is already out there in the wild and will be operating without any of us. And there could be many ways to connect to AEON though we only have one "aeonix" client as of now. This is a request to create a unique name to describe this client being developed here so that the word AEON can be used to describe what it really represents, a decentralized digital currency network. Also by using the right language, it may create room for other clients to be developed and co-exist.

**Comment by aeon1234** [comment]
this software (currently) is aeon.
maintaining more than one node software is extremely difficult, especially without documentation.
current implementation is basically also current documentation, so it's unlikely that we will ever see any other implementation without risking a network split. i would also consider something like this a waste of time at the moment.

but if you want to make another implementation, nobody will stop you from doing so.

**Comment by ghost** [comment]
I will consider that  :) Though I am not smart enough to manage the entire codebase by myself. The network need not split and many clients can peacefully coexist following the same rules similar that we have chrome and firefox. But neither firefox nor chrome would claim to be the internet itself.

**Comment by ghost** [comment]
By the way, it is as simple as referring to the software as aeonix in our language. This is more of a cultural language thing and less of a specific thing in the code that needs to be changed. Language can be a very valuable tool when used correctly. As an example "mining" blocks.

**Comment by aeon1234** [comment]
>The network need not split and many clients can peacefully coexist following the same rules

as I tried to explain, it's not that easy. every bug (eg diff calculation bug) in core implementation may split the network if this same bug is not in other implementations.

>By the way, it is as simple as referring to the software as aeonix

other cryptocurrencies sometimes refer to it as `core` implementation, so I will call this software `aeon` or `aeon core`.
`aeonix` is just the github username, I don't know why we should start calling the software `aeonix`. I also don't know what you try to achieve with this.

**Comment by ghost** [comment]
The achievement here is to help everyone realize that we could build something much bigger than just this one software application.. It could be AEON- the global **permissionless** currency network. That is much more exciting goal worth being a part of and captures the spirit. The achievement is to make distinction between AEON-(describing the p2p currency network) and this repository for software also named `aeon` which is used to connect to the network.  . 

`aeon core` would be sufficient but also boring and cliche.. I liked `aeonix` it is unique and futuristic 😎but others may disagree. Anything other than `aeon`. We have so much language why not make it useful instead of call everything aeon? This issue stands to correct that this type of language is used more commonly such as repo name, About, README, and in reference elsewhere.. etc..

I believe this is the right step to keep AEON permissionless and leaderless. Otherwise, we can just call aeon smoothcoin.

**Comment by BigslimVdub** [comment]
I agree with renaming the repo to ``aeon-core`` as the gui software is market “aeon-gui”. It does make sense. 

Just using “aeon” seems odd from a technical standpoint. 

Someone could ask “what is aeon” or “what does aeon mean” whereas with the GUI, very few would ask what “aeon-gui” is.
